### PR TITLE
Create CSS Entries for translateX & translateY and Update translate

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

Create new entries for docs/css/concepts/transform-functions/terms/translateX and docs/css/concepts/transform-functions/terms/translateX.

Updated some wording in docs/css/concepts/transform-functions/terms/translate

### Type of Change

- Adding a new entry
- Editing an existing entry 

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

File: content/css/concepts/transform-functions/terms/translateY/translateY.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Tags:
+  - 'Functions'
+  - 'Positioning'
+CatalogContent:
+  - 'learn-css'
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The `translateY` function translates an element along the Y axis.
```
Comment (yangc95) [inline content/css/concepts/transform-functions/terms/translateY/translateY.md:16]:
```suggestion
The `translateY` function translates an element along the vertical Y-axis.
```

File: content/css/concepts/transform-functions/terms/translateX/translateX.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+Tags:
+  - 'Functions'
+  - 'Positioning'
+CatalogContent:
+  - 'learn-css'
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The `translateX` function translates an element along the X axis.
```
Comment (yangc95) [inline content/css/concepts/transform-functions/terms/translateX/translateX.md:16]:
```suggestion
The `translateX` function moves an element along the horizontal X-axis.
```

File: content/css/concepts/transform-functions/terms/translate/translate.md [hunk 3]
```diff
@@ -13,24 +13,28 @@ CatalogContent:
   - 'paths/full-stack-engineer-career-path'
 ---
 
-Translates an element by one or more axes.
+The `translate` function translates an element along the X and/or Y axes.
```
Comment (yangc95) [inline content/css/concepts/transform-functions/terms/translate/translate.md:16]:
```suggestion
The `translate` function moves an element along the X- and/or Y-axis.
```

File: content/css/concepts/transform-functions/terms/translate/translate.md [hunk 4]
```diff
@@ -13,24 +13,28 @@ CatalogContent:
...
-transform: translate(<value>);
+```pseudo
+/* With a single value */
+transform: translate(<X-value>);
+
+/* With both values */
+transform: translate(<X-value>, <Y-value>);
 ```
 
 where a required `<value>` can be one of the following:
```
Comment (yangc95) [inline content/css/concepts/transform-functions/terms/translate/translate.md:28]:
```suggestion
A required `<X-value>` or `<Y-value` can be one of the following:
```

File: content/css/concepts/transform-functions/terms/translate/translate.md [hunk 5]
```diff
@@ -13,24 +13,28 @@ CatalogContent:
...
 
 - Length value: `100px`, `1.5em`
 - Percentage value: `25%`, `50%`
 
-**Note:** Providing only a single value will represent the horizontal axis. Providing a second comma separated value will represent the vertical axis. Values can be negative to translate elements in the opposite direction along the axes.
+Providing only a single value will represent the horizontal axis. Providing a second comma separated value will represent the vertical axis. A positive value moves the element right along the horizontal axis and down the vertical axis while a negative value moves the element left along the horizontal axis and up the vertical axis.
```
Comment (yangc95) [inline content/css/concepts/transform-functions/terms/translate/translate.md:33]:
```suggestion
Providing only a single value will represent the horizontal axis. Providing a second comma-separated value will represent the vertical axis. A positive value moves the element towards the right along the horizontal X-axis and down the vertical Y-axis. A negative value moves the element left along the horizontal X-axis and up the vertical Y-axis.
```

File: content/css/concepts/transform-functions/terms/translate/translate.md [hunk 6]
```diff
@@ -13,24 +13,28 @@ CatalogContent:
...
 
 ## Example 1
 
-Move the `.box` element `100px` along the horizontal axis:
+Move the `.box` element `100px` right along the horizontal axis:
```
Comment (yangc95) [inline content/css/concepts/transform-functions/terms/translate/translate.md:37]:
```suggestion
Move the element, with a `class` of `box`, `100px` right along the horizontal X-axis:
```

File: content/css/concepts/transform-functions/terms/translate/translate.md [hunk 7]
```diff
@@ -40,7 +44,7 @@ Move the `.box` element `100px` along the horizontal axis:
 
 ## Example 2
 
-Move the `.box` element along the horizontal axis `200px` and `100px` up the vertical axis:
+Move the `.box` element `200px` tight along the horizontal axis and `100px` up the vertical axis:
```
Comment (yangc95) [inline content/css/concepts/transform-functions/terms/translate/translate.md:47]:
```suggestion
Move the `.box` element `200px` along the horizontal X-axis and `100px` up the vertical Y-axis:
```

File: content/css/concepts/transform-functions/terms/translateX/translateX.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+```pseudo
+transform: translateX(<value>);
+```
+
+The required `<value>` can be one of the following:
+
+- Length value: `100px`, `1.5em`
+- Percentage value: `25%`, `50%`
+
+A positive value moves an element right along the horizontal axis while a negative value will move an element left.
```
Comment (yangc95) [inline content/css/concepts/transform-functions/terms/translateX/translateX.md:29]:
```suggestion
A positive value moves an element towards the right along the horizontal X-axis, while a negative value will move an element left.
```

File: content/css/concepts/transform-functions/terms/translateX/translateX.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+Note: `translateX(<value>)` is functionally equivalent to `translate(<value>)` when [`translate`](https://www.codecademy.com/resources/docs/css/transform-functions/transform) is called with a single value.
+
+## Example 1
+
+Move the `.box` element `100px` right:
```
Comment (yangc95) [inline content/css/concepts/transform-functions/terms/translateX/translateX.md:35]:
```suggestion
Move the element, with a `class` of `box`, `100px` to the right:
```

File: content/css/concepts/transform-functions/terms/translateY/translateY.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
+---
+Title: 'translateY()'
+Description: 'Translates an element along the Y axis.'
```
Comment (yangc95) [inline content/css/concepts/transform-functions/terms/translateY/translateY.md:3]:
```suggestion
Description: 'Translates an element along the Y-axis.'
```

File: content/css/concepts/transform-functions/terms/translateX/translateX.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
+---
+Title: 'translateX()'
+Description: 'Translates an element along the X axis.'
```
Comment (yangc95) [inline content/css/concepts/transform-functions/terms/translateX/translateX.md:3]:
```suggestion
Description: 'Translates an element along the X-axis.'
```

File: content/css/concepts/transform-functions/terms/translateY/translateY.md [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+```pseudo
+transform: translateY(<value>);
+```
+
+The required `<value>` can be one of the following:
+
+- Length value: `100px`, `1.5em`
+- Percentage value: `25%`, `50%`
+
+A positive value moves the element down the vertical axis while a negative value will move an element up.
```
Comment (yangc95) [inline content/css/concepts/transform-functions/terms/translateY/translateY.md:29]:
```suggestion
A positive value moves the element down the vertical Y-axis, while a negative value will move an element up.
```

File: content/css/concepts/transform-functions/terms/translateY/translateY.md [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+}
+```
+
+## Example 2
+
+Move the `.box` element `10em` up:
+
+```css
+.box {
+  transform: translateX(-10em);
```
Comment (yangc95) [inline content/css/concepts/transform-functions/terms/translateY/translateY.md:47]:
```suggestion
  transform: translateY(-10em);
```

**Review by yangc95 — COMMENTED** [review]
@SSwiniarski First review complete~

Assisted with making everything as explicit as possible and some other minor issues.

**Comment by SSwiniarski** [comment]
@yangc95, I implemented your suggestions.

**Comment by SSwiniarski** [comment]
@yangc95, I implemented your suggestions.

**Review by yangc95 — APPROVED** [review]
LGTM, ready for a second review!

File: content/css/concepts/transform-functions/terms/translate/translate.md [hunk 14]
```diff
@@ -13,24 +13,28 @@ CatalogContent:
...
+
+/* With both values */
+transform: translate(<X-value>, <Y-value>);
 ```
 
-where a required `<value>` can be one of the following:
+A required `<X-value>` or `<Y-value>` can be one of the following:
 
 - Length value: `100px`, `1.5em`
 - Percentage value: `25%`, `50%`
```
Comment (Dusch4593) [inline content/css/concepts/transform-functions/terms/translate/translate.md:31]:
```suggestion
```
I recommend moving this above the snippet.

File: content/css/concepts/transform-functions/terms/translate/translate.md [hunk 15]
```diff
@@ -13,24 +13,28 @@ CatalogContent:
   - 'paths/full-stack-engineer-career-path'
 ---
 
-Translates an element by one or more axes.
+The `translate` function moves an element along the X- and/or Y-axis.
 
 ## Syntax
```
Comment (Dusch4593) [inline content/css/concepts/transform-functions/terms/translate/translate.md:19]:
```suggestion

The required `<X-value>` or `<Y-value>` can be one of the following:

- Length value: `100px`, `1.5em`
- Percentage value: `25%`, `50%`

```

File: content/css/concepts/transform-functions/terms/translate/translate.md [hunk 16]
```diff
@@ -13,24 +13,28 @@ CatalogContent:
...
 
-```css
-transform: translate(<value>);
+```pseudo
+/* With a single value */
```
Comment (Dusch4593) [inline content/css/concepts/transform-functions/terms/translate/translate.md:21]:
```suggestion
/* Single value */
```

File: content/css/concepts/transform-functions/terms/translate/translate.md [hunk 17]
```diff
@@ -13,24 +13,28 @@ CatalogContent:
...
+transform: translate(<X-value>);
+
+/* With both values */
```
Comment (Dusch4593) [inline content/css/concepts/transform-functions/terms/translate/translate.md:24]:
```suggestion
/* Double values */
```

File: content/css/concepts/transform-functions/terms/translate/translate.md [hunk 18]
```diff
@@ -13,24 +13,28 @@ CatalogContent:
...
 ```
 
-where a required `<value>` can be one of the following:
+A required `<X-value>` or `<Y-value>` can be one of the following:
 
 - Length value: `100px`, `1.5em`
 - Percentage value: `25%`, `50%`
 
-**Note:** Providing only a single value will represent the horizontal axis. Providing a second comma separated value will represent the vertical axis. Values can be negative to translate elements in the opposite direction along the axes.
+Providing only a single value will represent the horizontal axis. Providing a second comma-separated value will represent the vertical axis. A positive value moves the element towards the right along the horizontal X-axis and down the vertical Y-axis. A negative value moves the element left along the horizontal X-axis and up the vertical Y-axis.
```
Comment (Dusch4593) [inline content/css/concepts/transform-functions/terms/translate/translate.md:33]:
```suggestion
A single value will represent only the horizontal axis. Providing a second comma-separated value will represent the vertical axis, as well. As as the direction goes:

* A positive value moves the element towards the right along the horizontal X-axis and down the vertical Y-axis (southeast). 
* A negative value moves the element left along the horizontal X-axis and up the vertical Y-axis (northwest).
```

File: content/css/concepts/transform-functions/terms/translate/translate.md [hunk 19]
```diff
@@ -13,24 +13,28 @@ CatalogContent:
...
 
 ## Example 1
 
-Move the `.box` element `100px` along the horizontal axis:
+Move the element, with a `class` of `box`, `100px` right along the horizontal X-axis:
```
Comment (Dusch4593) [inline content/css/concepts/transform-functions/terms/translate/translate.md:37]:
```suggestion
In the example below, an element with a `.box` class is moved `100px` to the right, along the horizontal X-axis:
```

File: content/css/concepts/transform-functions/terms/translate/translate.md (hunk 18 above)
Comment (Dusch4593) [reply content/css/concepts/transform-functions/terms/translate/translate.md:33]: Is there more to this with respect to moving in a northeast or southwest direction?

File: content/css/concepts/transform-functions/terms/translate/translate.md [hunk 20]
```diff
@@ -40,7 +44,7 @@ Move the `.box` element `100px` along the horizontal axis:
 
 ## Example 2
 
-Move the `.box` element along the horizontal axis `200px` and `100px` up the vertical axis:
+Move the `.box` element `200px` right along the horizontal X-axis and `100px` up the vertical Y-axis:
```
Comment (Dusch4593) [inline content/css/concepts/transform-functions/terms/translate/translate.md:47]:
```suggestion
In this example, the `.box` class element is moved `200px` to the right, along the X-axis, and `100px` up the vertical Y-axis.
```

File: content/css/concepts/transform-functions/terms/translateX/translateX.md [hunk 21]
```diff
@@ -0,0 +1,51 @@
+---
+Title: 'translateX()'
+Description: 'Translates an element along the X-axis.'
```
Comment (Dusch4593) [inline content/css/concepts/transform-functions/terms/translateX/translateX.md:3]:
```suggestion
Description: 'Moves an element horizontally along the X-axis.'
```

File: content/css/concepts/transform-functions/terms/translateX/translateX.md [hunk 22]
```diff
@@ -0,0 +1,51 @@
...
+Tags:
+  - 'Functions'
+  - 'Positioning'
+CatalogContent:
+  - 'learn-css'
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The `translateX` function moves an element along the horizontal X-axis.
```
Comment (Dusch4593) [inline content/css/concepts/transform-functions/terms/translateX/translateX.md:16]:
```suggestion
The `translateX()` function moves an element along the horizontal X-axis.
```

File: content/css/concepts/transform-functions/terms/translateX/translateX.md [hunk 23]
```diff
@@ -0,0 +1,51 @@
...
+```
+
+The required `<value>` can be one of the following:
+
+- Length value: `100px`, `1.5em`
+- Percentage value: `25%`, `50%`
+
+A positive value moves an element towards the right along the horizontal X-axis, while a negative value will move an element left.
+
+Note: `translateX(<value>)` is functionally equivalent to `translate(<value>)` when [`translate`](https://www.codecademy.com/resources/docs/css/transform-functions/transform) is called with a single value.
```
Comment (Dusch4593) [inline content/css/concepts/transform-functions/terms/translateX/translateX.md:31]:
```suggestion
**Note**: `translateX(<value>)` is functionally equivalent to `translate(<value>)` when [`translate`](https://www.codecademy.com/resources/docs/css/transform-functions/transform) is called with a single value.
```

File: content/css/concepts/transform-functions/terms/translateX/translateX.md [hunk 24]
```diff
@@ -0,0 +1,51 @@
...
+
+## Example 1
+
+Move the element, with a `class` of `box`, `100px` to the right:
```
Comment (Dusch4593) [inline content/css/concepts/transform-functions/terms/translateX/translateX.md:35]:
```suggestion
In the example below, an element with a `.box` class is moved `100px` to the right:
```

File: content/css/concepts/transform-functions/terms/translateX/translateX.md [hunk 25]
```diff
@@ -0,0 +1,51 @@
...
+
+```css
+.box {
+  transform: translateX(100px);
+}
+```
+
+## Example 2
+
+Move the `.box` element `10em` left:
```
Comment (Dusch4593) [inline content/css/concepts/transform-functions/terms/translateX/translateX.md:45]:
```suggestion
In this example, the element is moved to the left by `10em`, relative to the width of its direct parent:
```

File: content/css/concepts/transform-functions/terms/translateY/translateY.md [hunk 26]
```diff
@@ -0,0 +1,49 @@
+---
+Title: 'translateY()'
+Description: 'Translates an element along the Y-axis.'
```
Comment (Dusch4593) [inline content/css/concepts/transform-functions/terms/translateY/translateY.md:3]:
```suggestion
Description: 'Moves an element vertically along the Y-axis.'
```

File: content/css/concepts/transform-functions/terms/translateY/translateY.md [hunk 27]
```diff
@@ -0,0 +1,49 @@
...
+Tags:
+  - 'Functions'
+  - 'Positioning'
+CatalogContent:
+  - 'learn-css'
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The `translateY` function translates an element along the vertical Y-axis.
```
Comment (Dusch4593) [inline content/css/concepts/transform-functions/terms/translateY/translateY.md:16]:
```suggestion
The `translateY()` function moves an element along the vertical Y-axis.
```

File: content/css/concepts/transform-functions/terms/translateY/translateY.md [hunk 28]
```diff
@@ -0,0 +1,49 @@
...
+The required `<value>` can be one of the following:
+
+- Length value: `100px`, `1.5em`
+- Percentage value: `25%`, `50%`
+
+A positive value moves the element down the vertical Y-axis, while a negative value will move an element up.
+
+## Example 1
+
+Move the `.box` element `100px` down:
```
Comment (Dusch4593) [inline content/css/concepts/transform-functions/terms/translateY/translateY.md:33]:
```suggestion
In the example below, a `.box` element is moved down the Y-axis by `100px`:
```

File: content/css/concepts/transform-functions/terms/translateY/translateY.md [hunk 29]
```diff
@@ -0,0 +1,49 @@
...
+
+```css
+.box {
+  transform: translateY(100px);
+}
+```
+
+## Example 2
+
+Move the `.box` element `10em` up:
```
Comment (Dusch4593) [inline content/css/concepts/transform-functions/terms/translateY/translateY.md:43]:
```suggestion
In this example, the element is moved up the Y-axis by `-10em`, relative to the height of its direct parent element.
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Finished with my review of this one and left some comments/suggestions.

**Comment by SSwiniarski** [comment]
@Dusch4593, I implemented your suggestions and tried to clarify the northeast/southwest confusion.

File: content/css/concepts/transform-functions/terms/translate/translate.md [hunk 30]
```diff
@@ -13,24 +13,31 @@ CatalogContent:
...
-**Note:** Providing only a single value will represent the horizontal axis. Providing a second comma separated value will represent the vertical axis. Values can be negative to translate elements in the opposite direction along the axes.
+```pseudo
+/* Single value */
+transform: translate(<X-value>);
+
+/* Double values */
+transform: translate(<X-value>, <Y-value>);
+```
+
+A single value will represent only the horizontal axis. Providing a second comma-separated value will represent the vertical axis, as well. As as the direction goes:
```
Comment (Dusch4593) [inline content/css/concepts/transform-functions/terms/translate/translate.md:33]:
```suggestion
A single value will represent only the horizontal axis. Providing a second comma-separated value will represent the vertical axis, as well. As far as the direction goes:
```

**Comment by SSwiniarski** [comment]
@Dusch4593, I committed the last suggestion.

**Comment by yangc95** [comment]
Entries are live here:

- https://www.codecademy.com/resources/docs/css/transform-functions/translate
- https://www.codecademy.com/resources/docs/css/transform-functions/translateX
- https://www.codecademy.com/resources/docs/css/transform-functions/translateY

✨ Thanks to @SSwiniarski @Dusch4593

**Pull request merged**